test/vswhere.test/ExceptionsTests.cpp

```diff
@@ -28,15 +28,15 @@ TEST_CLASS(ExceptionsTests)
     {
         win32_error sut(ERROR_NOT_FOUND);
         Assert::AreEqual<int>(ERROR_NOT_FOUND, sut.code().value());
-        Assert::AreEqual("Element not found.\r\n", sut.what());
```

Comment:
This indicates some other potential problem. Characters don't magically disappear and tests were passing. What changed?

Reply:
When bumped to v143, there was a behavior change where message format for `system_error` does not add newline characters anymore. Since it's system api we don't own, I tried to match its behavior. I verified that the newlines are there when built with 141 but not 143.

Reply:
Okay. Since we don't control it that's fine. It was just a red flag moment since behavior changed albeit in an error message so I'm less concerned - if any - about behavior there since it's a failure case.